contents/newsletter/remote-working.md

```diff
@@ -0,0 +1,99 @@
+
+### 4. They make meeting up easy
+
+Pretty much all successful remote teams have one thing in common: they’re not remote all the time.
+
+That doesn’t mean hybrid working, or mandatory office days, it means both encouraging meeting up and making it easy to do so.
+
+At **PostHog**, we achieve this in a bunch of different ways:
+
+* **$300 monthly co-working budget**.** This makes it easy for people to meet up and work together if they already live in the same city.
+
+* **$1,500 quarterly meet-up budget** for ad hoc in-person collaboration with colleagues. This can cover flights, accommodation, and other expenses. At PostHog, Paul and Ben used this recently to meet for a week in Italy and ship our new [heatmap and scroll depth feature](https://posthog.com/changelog/2024#mouse-and-scroll-depth-tracking-launched).
+
+* **In-person onboarding.** When new people join the company, we prioritize organizing an in-person onboarding session as soon as possible. This helps people get to know each other and share important context.
+
+* **Annual company offsites.** The typical agenda includes a couple of big social events, team dinners, a hackathon, all-hands strategy sessions, company culture exercises, and downtime to socialize and explore.
+
+* **Frequent small team offsites.** In addition to our all-company offsite, all our [small teams](https://posthog.com/handbook/team-structure) run at least one annual offsite to focus on specific team goals and intense, in-person collaboration. They often include a hackathon, too, but this varies.
+
+Our goal with all of this is to make our in-person time as intentional as possible. To work on big, specific problems, build bonds that create trust, and make asynchronous work more creative and collaborative.
+
+### Good reads 📖
+
+**[Building an all-remote company from scratch](https://posthog.com/founders/remote-culture) – James Hawkins** This is exactly how we built a fully-remote company from the ground up. This was written in 2020 when we were 12 people. Today, we’re almost 50 and it’s still working.
+
+**[The manager’s schedule is holding remote work back](https://marker.medium.com/the-managers-schedule-is-holding-remote-work-back-f9c1302ac6f3) – Luke Thomas** Many makers (hello, product engineers!) have been working remotely successfully for years, but managers can often struggle. Luke shares how to fix that problem.
+
+**[Why remote work is so hard – and how it can be fixed](https://www.newyorker.com/culture/annals-of-inquiry/can-remote-work-be-fixed) – Cal Newport** This is a great, detailed read on the long-standing problems that have scuppered remote work. It’s written by Cal Newport, who also coined the term “deep work.”
```

Comment:
```suggestion
**[Why remote work is so hard – and how it can be fixed](https://www.newyorker.com/culture/annals-of-inquiry/can-remote-work-be-fixed) – Cal Newport** 
This is a great, detailed read on the long-standing problems that have scuppered remote work. It’s written by Cal Newport, who also coined the term “deep work.”
```